app/controllers/admin/api/services_controller.rb

```diff
@@ -100,7 +100,7 @@ def update
   ##~ op.parameters.add @parameter_service_id_by_id
   #
   def destroy
-    authorize! :destroy, service
+    authorize!(:destroy, service) if current_user
```

Comment:
Probably add a comment that this is OK since it is protected by the Authentication modules `lib/api_authentication/*`